doc/userguide/lua/libs/ssh.rst

```diff
@@ -0,0 +1,91 @@
+SSH
+---
+
+SSH transaction details are exposes to Lua scripts with the
+``suricata.ssh`` library, For example::
+
+  local ssh = require("suricata.ssh")
+
+Setup
+^^^^^
+
+If your purpose is to create a logging script, initialize the buffer as:
+
+::
+
+  function init (args)
+     local needs = {}
+     needs["protocol"] = "ssh"
+     return needs
+  end
+
+If you are going to use the script for rule matching, choose one of
+the available SSH buffers listed in :ref:`lua-detection` and follow
+the pattern:
+
+::
+
+  function init (args)
+     local needs = {}
+     needs["ssh.server_proto"] = tostring(true)
```

Comment:
Still valid with the change to hooks?

Reply:
Not really, fixing...

Should I test lua logging somehow ?

Reply:
Yeah, probably. Its expected to work. For example, here's an output script for DNS: tests/lua/lua-base64/output.lua

Reply:
Test added in https://github.com/OISF/suricata-verify/pull/2455 last commit